ember/app/ui/cases/detail/index/template.hbs

```diff
@@ -79,17 +79,35 @@
         <span>{{remark.value}}</span>
       </div>
     {{/each}}
-    {{#each this.permamentRemarks as |remark|}}
-      <span class="uk-text-bold uk-margin-bottom text-divider">
-        {{remark.workItem.document.form.name}}
-      </span>
-      {{#each remark.answers as |answer|}}
-        <div class="uk-margin-bottom">
-          <p class="uk-text-bold uk-margin-small-bottom">{{answer.label}}</p>
-          <span>{{answer.value}}</span>
+    <div class="uk-flex uk-flex-row">
```

Comment:
Should maybe only the value and date be in the flex and the comments full width again?